beekeeper-cleanup/src/test/java/com/expediagroup/beekeeper/cleanup/aws/S3ClientTest.java

```diff
@@ -207,6 +207,25 @@ void deleteObjectsInDirectory() {
     assertThat(amazonS3.doesObjectExist(bucket, key2)).isFalse();
   }
 
+  @Test
+  void deleteThousandObjectsInDirectory() {
+    ArrayList<String> keys = new ArrayList<>();
+    int totalObjects = 1000;
+    for(int i = 1; i <= totalObjects; i++){
+      var key = keyRoot + "/file" + i;
+      keys.add(key);
+    }
+    keys.parallelStream().forEach(key -> amazonS3.putObject(bucket, key, content));
+    int numberOfObjectsLeft = amazonS3.listObjects(bucket, keyRoot).getObjectSummaries().size();
+    assertThat(numberOfObjectsLeft).isEqualTo(totalObjects);
```

Comment:
i think these lines can be removed cause we don't need to test the s3 `putObject` functionality

Reply:
true, removed the assertion